go.mod

```diff
@@ -3,22 +3,8 @@ module github.com/olliefr/docker-gs-ping
 go 1.16
 
 require (
-	github.com/Microsoft/go-winio v0.5.0 // indirect
-	github.com/cenkalti/backoff v2.0.0+incompatible // indirect
-	github.com/containerd/continuity v0.1.0 // indirect
-	github.com/gotestyourself/gotestyourself v1.3.0 // indirect
-	github.com/konsorten/go-windows-terminal-sequences v1.0.3 // indirect
 	github.com/labstack/echo/v4 v4.2.2
-	github.com/mattn/go-colorable v0.1.8 // indirect
-	github.com/onsi/ginkgo v1.10.1 // indirect
-	github.com/onsi/gomega v1.7.0 // indirect
-	github.com/opencontainers/runc v1.0.0-rc93 // indirect
-	github.com/ory/dockertest/v3 v3.6.5 // indirect
-	github.com/sirupsen/logrus v1.8.1 // indirect
-	github.com/stretchr/testify v1.7.0
-	golang.org/x/crypto v0.0.0-20210421170649-83a5a9bb288b // indirect
-	golang.org/x/net v0.0.0-20210502030024-e5908800b52b // indirect
-	golang.org/x/sys v0.0.0-20210502180810-71e4cd670f79 // indirect
-	gopkg.in/airbrake/gobrake.v2 v2.0.9 // indirect
-	gopkg.in/gemnasium/logrus-airbrake-hook.v2 v2.1.2 // indirect
+	github.com/ory/dockertest/v3 v3.6.5
+	github.com/stretchr/testify v1.8.0
+	github.com/testcontainers/testcontainers-go v0.15.0
```

Comment:
Using previous version instead of latest 0.16.0 meanwhile we resolve the dependency issues from the native Docker Compose support we introduced in the latest release.